### PR TITLE
add intermediary loading boxes to profile image

### PR DESCRIPTION
**Pull request by remytruong**

### CODE REVIEW

File: components/d2l-profile-image.js [hunk 1]
```diff
@@ -15,11 +15,13 @@ import 'd2l-polymer-siren-behaviors/store/siren-action-behavior.js';
...
 import './d2l-profile-image-base.js';
 import { Polymer } from '@polymer/polymer/lib/legacy/polymer-fn.js';
+import './styles/d2l-profile-image-shared-styles.js';
+
 const $_documentContainer = document.createElement('template');
 
 $_documentContainer.innerHTML = `<dom-module id="d2l-profile-image">
 	<template strip-whitespace="">
-			<style>
+			<style include="d2l-profile-image-shared-styles">
```
Comment (svanherk) [inline components/d2l-profile-image.js:24]: I'm surprised this was needed, since the `loading` property is passed down to the base image component 🤔

**Comment by svanherk** [comment]
Can you add a super simple example of a loading component to the demo page while you're here?

File: components/d2l-profile-image.js (hunk 1 above)
Comment (remytruong) [reply components/d2l-profile-image.js:24]: Ah, sorry I forgot to remove this - I was iterating on a different implementation and needed the other css... I've deleted!

**Review by svanherk — APPROVED** [review]
Thanks!

**Comment by svanherk** [comment]
Do you have permission to merge?  To release, you'll want to add "feat: " to the beginning of your merge commit

**Comment by remytruong** [comment]
I do!

**Pull request merged**